lasagne/init.py

```diff
@@ -8,14 +14,48 @@
 
 
 class Initializer(object):
+    """Initializer class
+
+    The :class:`Initializer` class represents a weight initializer used
+    to initialize weight parameters in a neural network layer. It should be
+    subclassed when implementing new types of weight initializers.
+
+    """
     def __call__(self, shape):
+        """
+           Subclasses of :class:`Layer` class use this method when they
+           initialize weight parameters.
```

Comment:
Intended by three spaces too much, and `Layer` is wrong here :) I'd instead write:
```
Makes :class:`Initializer` instances callable like a function, invoking their :meth:`sample()` method.
```
(Is that the correct markup?)